infrastructure/template.yaml

```diff
@@ -2194,6 +2194,36 @@ Resources:
       ComparisonOperator: GreaterThanOrEqualToThreshold
       TreatMissingData: notBreaching
 
+  NinoCheckFunction5xxCanaryErrors:
+    Type: AWS::CloudWatch::Alarm
+    Condition: UseCanaryDeploymentAlarms
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      OKActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      AlarmDescription: !Sub "Check HRMC NINO Check Lambda returning 5xx response."
+      AlarmName: !Sub ${AWS::StackName}-${Environment}-NinoCheckFunction5xxCanaryErrors
+      Namespace: AWS/ApiGateway
+      MetricName: 5XXError
+      Dimensions:
+        - Name: ApiName
+          Value: !Sub "check-hmrc-cri-api-private-${AWS::StackName}"
```

Comment:
`!Sub ${AWS::StackName}-private` I think this should point to the name attribute we used for `PrivateNinoCheckApi`